src/sender.js

```diff
@@ -7,15 +7,12 @@ const Pathfinder = require('five-bells-pathfind').Pathfinder
 function Sender (params) {
   this.source_ledger = params.source_ledger
   this.source_username = params.source_username
+  this.source_account = params.source_account || toAccount(this.source_ledger, this.source_username)
   this.source_password = params.source_password
   this.destination_ledger = params.destination_ledger
-  this.destination_username = params.destination_username
```

Comment:
Arguably dealing only with the `destination_account` is better because we shouldn't be manipulating these URIs anyway. That would technically be a major change I guess